### DISCUSSION TITLE
Apache Solr

### DISCUSSION THREAD

**Discussion by AruneshSingh** [superlinked/VectorHub#78] [VDB Comparison]
Hey everyone, this is the start of discussions related to [Apache Solr](https://solr.apache.org/guide/solr/latest/query-guide/dense-vector-search.html#usage-as-re-ranking-query) and their attributes as compared in the table [here](https://vdbs.superlinked.com). 

To add to the discussion please check if there's already a comment with the attribute you want to discuss about, and reply in the thread. Otherwise please create a new comment for the attribute you want to discuss about.

To create an issue, please have a look at the [README](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/README.md) for contributing guidelines.

Let's all try to have a fruitful discussion!

**Comment by alessandrobenedetti** [comment]
In regards to "in process" I think we need a green flag for Apache Solr through the EmbeddedSolrServer:
https://solr.apache.org/docs/9_4_0/core/org/apache/solr/client/solrj/embedded/EmbeddedSolrServer.html

**Reply by svonava** [reply]
Can you make a PR for it? And please don't forget to link the source as well! 🙏

**Reply by alessandrobenedetti** [reply]
done
